lib/Dialect/SPIRV/Serialization/Deserializer.cpp

```diff
@@ -722,6 +749,46 @@ LogicalResult Deserializer::processFunctionType(ArrayRef<uint32_t> operands) {
   return success();
 }
 
+LogicalResult Deserializer::processStructType(ArrayRef<uint32_t> operands) {
+  if (operands.size() < 2) {
+    return emitError(unknownLoc, "OpTypeStruct must have at least one member");
+  }
+
+  SmallVector<Type, 0> memberTypes;
+  for (uint32_t opNum = 1; opNum < operands.size(); ++opNum) {
```

Comment:
I think we can do `for (auto member : operands)` here?

Reply:
Thanks, seems like I can fix it like this:
`for (auto member :make_range(operands.begin() + 1, operands.end())`

Reply:
llvm::drop_begin(operands, 1)

Reply:
Oh, thanks!